internal/controller/flow_falco.go

```diff
@@ -36,6 +38,29 @@ func removeTrailingTab(time string) string {
 	return strings.TrimRight(time, "\t")
 }
 
+// convertStringToTimestamp converts the provided string value to a timestamppb.Timestamp object
+// and updates the Time field of the FalcoEvent struct with the provided value.
+// It parses the input string value in RFC3339 format and returns a timestamppb.Timestamp object.
+func convertStringToTimestamp(value string) (*timestamppb.Timestamp, error) {
```

Comment:
1. If Falco's format is really what you specify below, this is not strictly speaking RFC 3339. Clarify that.
2. This function does not update any field.
```suggestion
```
```suggestion
// parseFalcoTimestamp  parses the input string value in a Falco's timestamp format that is similar to RFC3339 and returns a timestamppb.Timestamp.
func parseFalcoTimestamp(value string) (*timestamppb.Timestamp, error) {
```